scanpy/plotting/_tools/__init__.py

```diff
@@ -365,7 +365,7 @@ def _fig_show_save_or_axes(plot_obj, return_fig, show, save):
         plot_obj.make_figure()
         savefig_or_show(plot_obj.DEFAULT_SAVE_PREFIX, show=show, save=save)
         show = settings.autoshow if show is None else show
-        if not show:
+        if show == False:  # noqa: E712  # TODO Does this need fixing? -> is False
```

Comment:
I'd be fine with this:

```suggestion
        if show is False:
```

Reply:
I think `flake8` allows this, but pep8 doesn't. Not sure if this would create conflicts with `autopep8`.